project.clj

```diff
@@ -57,46 +72,36 @@
                  ;; EDN reader with location metadata - for configuration
                  [org.clojure/tools.reader "0.8.8"]
 
-                 [liberator "0.12.2"]
-                 [cheshire "5.3.1"]
+                 ;; ClojureScript dependencies
 
-                 ;; Required for Cassandra (possibly only OSX)
-                 [org.xerial.snappy/snappy-java "1.1.1.3"]
+                 [org.clojure/clojurescript "0.0-2665" :scope "provided"]
+                 [org.clojure/core.async "0.1.346.0-17112a-alpha" :scope "provided"]
```

Comment:
Is this the correct version pairing for cljs and core.async? I vaguely remember seeing some issues around this so I just want to check.